R/backend-sqlite.R

```diff
@@ -141,14 +141,31 @@ sql_query_join.SQLiteConnection <- function(con, x, y, vars, type = "inner", by
   # workaround as SQLite doesn't support FULL OUTER JOIN and RIGHT JOIN
   # see: https://www.sqlite.org/omitted.html
 
-  # Careful: in the right join resp. the second join in the full join do not
-  # name `y` LHS and `x` RHS as it messes up the select query!
+  if (!type %in% c("full", "right")) {
```

Comment:
I think it's better to be explicit about which joins it applies to, not which joins it doesn't.